samples/README.md

```diff
@@ -9,112 +9,74 @@ Test the SDK by adding `VAULT-ID`, `VAULT-URL`, and `SERVICE-ACCOUNT` details in
 
 ## Prepare
 
-- Install the Node SDK:
+Install the Node SDK:
 
 ```bash
 npm install skyflow-node
 ```
 
 ### Create a vault
 
-1. In a browser, navigate to Skyflow Studio.
-2. Create a vault by clicking **Create Vault** > **Start With a Template** > **Quickstart vault**.
-3. Once the vault is created, click the gear icon and select **Edit Vault Details**.
-4. Note your **Vault URL** and **Vault ID** values, then click **Cancel**. You'll need these later.
+1. Navigate to Skyflow Studio in your browser.
+2. Click **Create Vault** > **Start With a Template** > **Quickstart vault**.
+3. When the vault is ready, click the gear icon and select **Edit Vault Details**.
+4. Copy your **Vault ID** and **Cluster ID** from the URL (format: `https://{clusterId}.vault.skyflowapis.com`), then click **Cancel**.
 
 ### Create a service account
 
-1. In the side navigation click, **IAM** > **Service Accounts** > **New Service Account**.
-2. For **Name**, enter "SDK Samples". For **Roles**, choose **Vault Editor**.
-3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure. You'll need it for each of the samples.
+1. Click **IAM** > **Service Accounts** > **New Service Account** in the side navigation.
```

Comment:
Not new, but 'IAM' does not exist in Studio. Maybe we remove the Studio directions and link to docs.skyflow.com @manny-silva ?